src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,18 @@
+.footer {
+  height: 180px;
+  background-image: url("/image/footer.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 100%;
+
+  z-index: -1;
+
+  @include onTablet {
+    height: 360px;
+  }
+
+  @include onDesktop {
+    height: 360;
```

Comment:
The height value is missing a unit (e.g., px or rem). Please add the appropriate unit.